modules/sdk/gradle-plugins-osgi-bundle-builder/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: "org.ysb33r.gradletest"
+
+//apply plugin: "com.gradle.plugin-publish"
+
+task copyGradleTestDependencies(type: Copy)
+
+copyGradleTestDependencies {
+	from configurations.compile
+	into jar.destinationDir
+}
+
+dependencies {
+	compile group: "com.liferay", name: "com.liferay.gradle.util", version: "1.0.29"
+
+	compileOnly fileTree(builtBy: [rootProject.tasks.extractGradleApi25], dir: new File(rootProject.buildDir, "gradle-2.5"))
+}
+
+gradleTest {
```

Comment:
You added this code, but I don't see any Gradle tests. Can you please add a *smoke* test? Thanks!